### PR TITLE
OD-609 [Fix] Correctly show a pointer cursor on inline links

### PR DESCRIPTION
**Pull request by skrupnyk**
@romanyosyfiv @inna-bieshulia

OD-609 https://weboo.atlassian.net/browse/OD-609

Description
**Problem:** Text links do not have a default pointer cursor because the desired styles are not specified for which the pointer should change.
**Solution:** Added styles that change cursor to pointer when hovering over a text link.

Screenshots/screencasts
https://storyxpress.co/video/kunzpzvym9gmznpfu

Backward compatibility
This change is fully backward compatible.

Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: css/build.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.inline-link {
```
Comment (tonytlwu) [inline css/build.css]: You don't need to add a new class to the link. The widget has a container that provides sufficient context. It also means developers can have an element with a custom `inline-link` class without being affected.

```suggestion
[data-widget-package="com.fliplet.inline-link"] > a {
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "widget-inline-link",
+  "version": "1.0.0",
+  "description": "Documentation is available at https://developers.fliplet.com/API/components/form-builder.html",
```
Comment (tonytlwu) [inline package.json]: Incorrect reference

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: css/build.css (hunk 1 above)
Comment (skrupnyk) [reply css/build.css]: fixed

**Review by tonytlwu — APPROVED** [review]
@romanyosyfiv @inna-bieshulia This is ready for testing

Risk: Low
Affected: Inline link users in apps
Verify: Users can click on the inline links

Squash and merge: Use PR title

**Comment by tonytlwu** [comment]
@upplabs-alex-levchenko Feedback for the team: the PR title was "Ensures hidden fields in forms are not displayed to users" which looks like it's for an entirely different PR.

**Pull request merged**